CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ Those changes in added, changed or breaking changes, should include usage exampl
 
 ## [Unreleased]
 ### Added
+### Changed
+### Breaking changes
+### Removed
+
+## [0.2.0] - 2025-02-06
```

Comment:
Can we sync this changelog with [the one in draft](https://github.com/Layr-Labs/eigensdk-rs/releases/tag/untagged-70261fda40cc74afb73b)?